3-UsingAPIs/Week2/homework/ex3-rollAnAce.js

```diff
@@ -10,24 +10,24 @@
 // ! Do not change or remove the next two lines
 const rollDice = require('../../helpers/pokerDiceRoller');
 
-function rollDiceUntil(wantedValue) {
-  // TODO: rewrite this function using async/await
-  const recurse = () => {
-    return rollDice().then((value) => {
-      if (value !== wantedValue) {
-        return recurse();
-      }
-      return value;
-    });
-  };
-  return recurse();
+async function rollDiceUntil(wantedValue) {
+  //TODO: rewrite this function using async/await
+  let value = await rollDice()
+  while (value !== wantedValue){
+     value = await rollDice()
+  }
+  return value
```

Comment:
Perfect! Much easier to work with `async` and `await` than with standard promise syntax, right?